miio/toiletlid.py

```diff
@@ -139,9 +136,13 @@ def set_ambient_light(self, color: AmbientLightColor):
 
     @command(default_output=format_output("Get the Ambient light color."))
     def get_ambient_light(self) -> str:
-        """Set Ambient light color."""
+        """Get Ambient light color."""
         color = self.send("get_aled_v_of_uid", [""])
-        if color:
+        try:
             return AmbientLightColor(color[0]).name
-        else:
-            return AmbientLightColor("0").name
+        except TypeError:
+            raise DeviceException(
+                "Get Ambient Light Operating Device Not Responding!"
+            ) from None
+        except ValueError as e:
+            raise AmbientLightException(e) from None
```

Comment:
Is there really need to handle this case (and to that end, is it even necessary to have a custom exception for this)? How often can this happen? I think it would make more sense to check for the value of `color[0]` and raise an exception on unexpected value if that's necessary at all.